src/workerd/api/pyodide/pyodide.c++

```diff
@@ -343,13 +349,32 @@ kj::Array<kj::StringPtr> ArtifactBundler::getSnapshotImports() {
   return kj::heapArray(snapshotImports.begin(), snapshotImports.size());
 }
 
-bool isWorkerModuleImport(
-    kj::HashSet<kj::String>& workerModules, kj::ArrayPtr<const char> firstComponent) {
+// This is equivalent to `pkgImport.replace('.', '/') + ".py"`.
+kj::String importToModuleFilename(kj::StringPtr pkgImport) {
+  auto result = kj::heapString(pkgImport.size() + 3);
+  for (auto i = 0; i < pkgImport.size(); i++) {
+    if (pkgImport[i] == '.') {
+      result[i] = '/';
+    } else {
+      result[i] = pkgImport[i];
+    }
+  }
```

Comment:
Nit:

Instead of copying one by one, we can copy the whole string and later to a replace on the character which would make this code a little bit more readable.

I'm not sure if there is a kj method for this:


```
std::replace(s.begin(), s.end(), '.', '/');
```